cas-server-support-x509/pom.xml

```diff
@@ -54,10 +54,10 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <artifactId>vt-crypt</artifactId>
-      <groupId>edu.vt.middleware</groupId>
-      <version>2.1.4</version>
-    </dependency>
+	  <groupId>org.cryptacular</groupId>
```

Comment:
Yes, but if someone wants to use it, he will find it in the root pom while he would'nt know if it's only defined in this pom.